syslog_ng/files/syslog-ng.conf

```diff
@@ -16,9 +16,17 @@ def rule_builder(rule):
       return rule[1:]
     return '"%s"' % rule
 %>\
-@version: 3.3
 
-@module tfjson
+@version: ${ syslog_config.get('version', '3.3') }
+
+% for module in syslog_config.get('module', ['tfjson']):
```

Comment:
Can you please change it from 
```
['tfjson']
```
to 
```
[]
```
We don't want an unintentional feature install for people.